gulpfile.babel.js

```diff
@@ -1,18 +1,40 @@
 import gulp from 'gulp';
 import requireDir from 'require-dir';
-import livereload from 'gulp-livereload';
+import browserSync from 'browser-sync';
 
 requireDir( './gulp-tasks' );
 
+const packageJson = require('../../../package.json');
+const localURL = packageJson.testing.urls.local;
```

Comment:
@colorful-tones this is a testing URL, we probably can't piggyback off it for this purpose. Better to use a new object from the package file. maybe just `package.localUrl`?